Creamy tomato soup, soup-free chili.

Dorsisdwarf opened the pull request:
i tried to make chili and it made 50% tomato soup immersion RUINED
## About The Pull Request
Adds 5u of cream to the recipe for tomato soup. This means that tomatosoup is no longer produced as an unavoidable byproduct of all soup recipes that include a tomato when making more than one serving.
## Why It's Good For The Game
Fixes #75886 , makes soup tastier
## Changelog
:cl:
fix: Nanotrasen Cookery Command has ended the plague of staff complaining that their tomato dishes are just complicated forms of soup by redefining "tomato soup" to include a dash of cream.
/:cl:

Dorsisdwarf committed:
Adds cream to tomato soup recipie

see issue 75886!

DaydreamIQ commented:
Glad im not the only one who had this exact issue. Mah fuckin chilli was RUINED

CRITAWAKETS commented:
Not every tomato soup uses cream (please help my dad sucks at making tomato soup), since there isn't any basil in the game, I think maybe black pepper could be an interesting ingredient instead?
plus, cream in tomato soup makes it non-vegan, which is kinda a weird property for tomato soup to have

Fikou commented on code/modules/food_and_drinks/recipes/soup_mixtures.dm:
```diff
@@ -551,7 +551,10 @@
 /datum/chemical_reaction/food/soup/tomatosoup
-	required_reagents = list(/datum/reagent/water = 50)
+	required_reagents = list(
+		/datum/reagent/water = 50,
```
```suggestion
		/datum/reagent/water = 45,
```

Dorsisdwarf replied:
Given that you always put 50u of water in a cookpot when you fill it this is just going to end up with leftover water in the soup mix, but I'll do it.

Dorsisdwarf committed:
reduces water in tomato soup 

As requested, reduces water in tomato soup to 45u

Dorsisdwarf committed:
undoes previous commit

it was decided that 50u of water was more suitable than 45u to reduce wastewater with standard pot filling sizes

Sealed101 commented:
don't tell goof
